README.md

```diff
@@ -1,5 +1,6 @@
 # Fundamental React
 
+[![REUSE status](https://api.reuse.software/badge/github.com/SAP/fundamental-react)](https://api.reuse.software/info/github.com/SAP/fundamental-react)
```

Comment:
@droshev fundamental@sap.com shoudl have recieved an email to register our project to use this badge.

Reply:
https://api.reuse.software/info/github.com/SAP/fundamental-react We failed lol